stl/inc/xstring

```diff
@@ -5192,11 +5207,10 @@ using u16string = basic_string<char16_t, char_traits<char16_t>, allocator<char16
 using u32string = basic_string<char32_t, char_traits<char32_t>, allocator<char32_t>>;
 
 template <class _Elem, class _Traits, class _Alloc>

```

Comment:
Similarly, we should remove the `class _Traits` template parameter here and replace occurrences with `char_traits<_Elem>`.